Tasks/AzurePowerShellV4/Utility.ps1

```diff
@@ -25,6 +25,30 @@ function Update-PSModulePathForHostedAgent {
         else {
             $hostedAgentAzModulePath = Get-LatestModule -patternToMatch "^az_[0-9]+\.[0-9]+\.[0-9]+$" -patternToExtract "[0-9]+\.[0-9]+\.[0-9]+$"
         }
+        
+        $azureRMModulePath = "C:\Modules\azurerm_2.1.0"
```

Comment:
Over time, this function has become complicated. I think, we should move all of this into the pre job execution handler. Perhaps we can perform validations and path updates there and use the result here in the main script task.

Reply:
Also, once you update the psmodulepath to have only one version, you do not need to import az module every time. It should work even without importing the module. This should improve the task execution time quite a bit.

Reply:
It is done.